dataeng/jobs/analytics/TableauConfigBackup.groovy

```diff
@@ -0,0 +1,52 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+class TableauConfigBackup{
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("tableau-config-backup"){
+            logRotator common_log_rotator(allVars)
+            parameters secure_scm_parameters(allVars)
+            multiscm secure_scm(allVars)
+            parameters {
+                stringParam('ANALYTICS_TOOLS_URL', allVars.get('ANALYTICS_TOOLS_URL'), 'URL for the analytics tools repo.')
+                stringParam('ANALYTICS_TOOLS_BRANCH', allVars.get('ANALYTICS_TOOLS_BRANCH'), , 'Branch of analytics tools repo to use.')
+                stringParam('TABLEAU_SERVER_IP', allVars.get('TABLEAU_SERVER_IP'), 'IP address of Tableau Server.')
+                stringParam('USER_NAME', allVars.get('USER_NAME'), 'SSH User name.')
+                stringParam('S3_BUCKET', allVars.get('S3_BUCKET'), 'S3 backup bucket name with path.')
```

Comment:
For the last few params, can you give a more helpful description of what the parameter is used for?

Reply:
Fixed it.